packages/nextjs/liveblocks.config.ts

```diff
@@ -0,0 +1,52 @@
+import { createClient } from "@liveblocks/client";
+import type { BaseUserMeta } from "@liveblocks/client";
+import { createRoomContext } from "@liveblocks/react";
+
+const client = createClient({
+  throttle: 16,
+  publicApiKey: process.env.NEXT_PUBLIC_LIVEBLOCKS_PUBLIC_KEY!,
```

Comment:
you'll need to add this api key to vercels deployment secrets

Reply:
![image](https://github.com/jaxonL/copix/assets/60107091/49a45dd1-c972-4a4c-8245-cb206f912707)
Yup